src/app/components/top/solicitudes/solicitudes.component.ts

```diff
@@ -243,6 +243,30 @@ export class SolicitudesComponent implements OnInit {
         return this.auth.check('rup:tipoPrestacion:' + prestacion.solicitud.tipoPrestacion.id);
     }
 
+    // check if the shift type is virtual and store the result in the model so as not to check the database again
+    isVirtualTurn(prestacion) {
+        let link = '';
+        if (prestacion.solicitud.actualizado === undefined) {
+            prestacion.solicitud.actualizado = false;
+            if (prestacion.solicitud?.turno) {
+                this.servicioTurnos.getTurnos({ id: prestacion.solicitud.turno }).subscribe(turnos => {
+                    link = turnos[0].bloques[0].turnos[0].link;
+                    if (link !== undefined) {
```

Comment:
`null`  o `''` son valores válidos?
```suggestion
                    if (link) {
```